examples/cpp/RecordReplay/holistic_record.cpp

```
@@ -9,58 +11,64 @@
     #error This example needs OpenCV support, which is not available on your system
 #endif
 
+// Signal handling for clean shutdown
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)